rust/patchable/src/main.rs

```diff
@@ -474,116 +496,167 @@ fn main() -> Result<()> {
             );
         }
 
-        Cmd::Init {
-            pv,
-            base,
-            mirror,
-            ssh,
-        } => {
-            let ctx = ProductVersionContext {
-                pv,
-                images_repo_root,
-            };
+        Cmd::Init { init_type } => match init_type {
+            InitType::Product {
+                product,
+                upstream,
+                default_mirror,
+            } => {
```

Comment:
```suggestion
        Cmd::Init { init_type: InitType::Product {product, upstream, default_mirror} } => {
```
No need to nest the match blocks if there's no code in common between the branches anyway.

Reply:
Fixed in https://github.com/stackabletech/docker-images/pull/1112/commits/c623c3178d803238daf70ee5b9c32969db67d62a